pulp_rpm/tests/functional/utils.py

```diff
@@ -79,83 +79,16 @@ def get_rpm_package_paths(repo):
     ]
 
 
-def populate_pulp(cfg, url=RPM_SIGNED_FIXTURE_URL):
-    """Add RPM contents to Pulp.
+def gen_rpm_content_attrs(artifact, rpm_name):
+    """Generate a dict with content unit attributes.
 
-    :param pulp_smash.config.PulpSmashConfig: Information about a Pulp
-        application.
-    :param url: The RPM repository URL. Defaults to
-        :data:`pulp_smash.constants.RPM_UNSIGNED_FIXTURE_URL`
-    :returns: A list of dicts, where each dict describes one RPM content in
-        Pulp.
+    :param artifact: A dict of info about the artifact.
+    :returns: A semi-random dict for use in creating a content unit.
     """
-    client = api.Client(cfg, api.json_handler)
-    remote = {}
-    repo = {}
-    try:
-        remote.update(client.post(RPM_REMOTE_PATH, gen_rpm_remote(url)))
-        repo.update(client.post(RPM_REPO_PATH, gen_repo()))
-        sync(cfg, remote, repo)
-    finally:
-        if remote:
-            client.delete(remote['pulp_href'])
-        if repo:
-            client.delete(repo['pulp_href'])
-    return client.get(RPM_CONTENT_PATH)['results']
-
-
-def gen_yum_config_file(cfg, repositoryid, baseurl, name, **kwargs):
```

Comment:
is it not needed anymore?

Reply:
anywhere used, so no need to keep it

Reply:
maybe it is used on jenkins,
@rochacbruno could you confirm if QE uses this?

Reply:
@fabricio-aguiar we can remove it as it is not being used on this repo, we dont have other codebases for pulp3 tests, and this scenario is going to be covered downstream later.

Reply:
I would prefer any outright removals go into a separate commit.

Reply:
lets keep this one